ls1intum/Artemis
# Jenkins Buildlogs not being shown

### Checklist
- [] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context
Build logs could not be fetches for jenkins, see #1436 

### Description
There was an internal error being thrown, because a wrong ZoneDateTimeFormat was inserted.
The buildlogs provided by jenkins also contain the logs for the build pipeline. Therefore, i pruned the buildlogs to only provide interesting logs to the student.

### Steps for Testing
1. Log in to artemistest2.ase.in.tum.de
2. Create a **Java** exercise
3. Enable the Code Editor
4. Launch the Exercise, and submit a build 
5. The build logs should be shown 

### Screenshots
![image](https://user-images.githubusercontent.com/15110960/82562661-fb6e2500-9b75-11ea-9170-98b40b48c196.png)
